src/app.js

```diff
@@ -1 +1,76 @@
-console.log('Hallo daar!');
\ No newline at end of file
+import axios from "axios";
+
+const countryList = document.getElementById("countries")
+async function fetchData() {
+    try {
+    const response = await axios.get("https://restcountries.com/v3.1/all?fields=name,flags,population,continents");
+
+
+    /*      for (i = 0; i < response.data; i++) {}        */
+    /*      response.map((country) => {})                 */
+
+    const countries = response.data;
+    allCountries(response);
+    buttons(countries);
+    } catch (e) {
+        console.error(e)
+    }
+}
+
+fetchData();
+
+function allCountries(countries) {
+    countryList.innerHTML =
+    countries.data.map((country) => {
+        return `<li>
+        <div class="landBox">
+            <img src="${country.flags.png}" alt="this is the flag of ${country.name.common}"/>
+                <div class="populationBox">
+                    <h3 class="${fetchRegion(country.continents[0])}">${country.name.common}</h3>
+                    <p>Has a population of ${country.population}</p>
+                </div> 
+            </div>
+        </li>`;
+    })
+        .join("");
```

Comment:
Als je de innerHTML in je .map methode uitvoert met += dan hoef je hier niet .join toe te passen, uiteindelijk is het resultaat het zelfde alleen hoef je dan wat minder typwerk uit te voeren.

Reply:
als ik dit weghaal komt er tussen elk item een komma.

Reply:
als je nog wilt kunnen we er wel een keer samen naar kijken